packages/slate/src/transforms/general.ts

```diff
@@ -65,7 +65,8 @@ export const GeneralTransforms = {
         if (Text.isText(node) && Text.isText(prev)) {
           prev.text += node.text
         } else if (!Text.isText(node) && !Text.isText(prev)) {
-          prev.children.push(...node.children)
+          const children = node.children as Element[]
```

Comment:
I changed the type of `editor.children` to `Descendant[]` instead of `Element[]` and am going to revert some of these changes.  We can discuss but I think the more appropriate type for `editor.children` is `Descendant[]`.

Reply:
okay! I'll wait for you push then